tldr/dfg_extractor.py

```
@@ -3738,3 +3745,169 @@ def extract_luau_dfg(code: str, function_name: str) -> DFGInfo:
+        for child in node.children:
+            if child.type not in ("identifier", "const", "var", "type_expression", "primary_type_expression"):
+                self._visit_node(child)
+
+        for child in node.children:
+            if child.type == "identifier":
+                name = self.get_node_text(child)
+                if self._is_valid_var_name(name):
+                    self._add_ref(name, "definition", child)
+                break
+
+    def _handle_assignment(self, node):
```

Ruff (0.14.14)

Missing return type annotation for private function `_handle_assignment`

Add return type annotation: `None`

(ANN202)